packages/app/src/client/views/ring-health/RingTable.tsx

```diff
@@ -78,6 +78,7 @@ const RingTable = ({ ringEndpoint, baseUrl }: Props) => {
     registerNotification,
     unregisterNotification
   } = useNotificationService();
+  const [keepPolling, setKeepPolling] = useState(true);
```

Comment:
I added this to interrupt the polling when errors happen. 

Otherwise we just add a bunch of notifications to the UI for each subsequent failure of fetching.